tella-locking-ui/src/main/java/com/hzontal/tella_locking_ui/common/util/DivviupUtils.kt

```diff
@@ -5,23 +5,32 @@ import com.hzontal.tella_locking_ui.R
 import org.divviup.android.Client
 import org.divviup.android.TaskId
 import org.hzontal.shared_ui.data.CommonPreferences
+import org.hzontal.tella.keys.util.Preferences
 import timber.log.Timber
 import java.net.URI
 import java.util.concurrent.Executors
 
 object DivviupUtils {
```

Comment:
I would suggest to refactor this to a Non-object Class :
This allows you to inject dependencies, like `Context` and the `Executors` more cleanly ..

Reply:
it could be injected like this -> 
```
@Singleton
class DivviupUtils @Inject constructor(
// what ever we need to provide
```